NoNicotine_Business/Handler/CreatePatientCommandHandler.cs

```diff
@@ -23,14 +23,16 @@ public class CreatePatientCommandHandler : IRequestHandler<CreatePatientCommand,
 
         private readonly UserManager<IdentityUser> _userManager;
         private readonly ILogger<CreatePatientCommandHandler> _logger;
+        private readonly IPatientRepository _patientRepository;
         private readonly AppDbContext _context;
         private const string PATIENT_ROLE = "patient";
-        public CreatePatientCommandHandler( UserManager<IdentityUser> userManager, 
-            ILogger<CreatePatientCommandHandler> logger, AppDbContext dbContext)
+        public CreatePatientCommandHandler( UserManager<IdentityUser> userManager,
+            ILogger<CreatePatientCommandHandler> logger, IPatientRepository patientRepository, AppDbContext context)
         {
             _userManager = userManager;
             _logger = logger;
-            _context = dbContext;
+            _patientRepository = patientRepository;
+            _context = context;
```

Comment:
Why do you have both context and repository here, but in other classes only have repository (f.e. GetPatientQueryHandler)??